### ISSUE TITLE
Retrieve container parameters in TraaS

### ISSUE THREAD

**Issue by YenKang** [eclipse/sumo#5542]
Dear @behrisch , @namdre 

After reading the [specification/logistic](https://sumo.dlr.de/wiki/Specification/Logistics) and [Specification/Containers](https://sumo.dlr.de/wiki/Specification/Containers), I successfully set up the ***containerStop***

![containerStop](https://user-images.githubusercontent.com/16321107/57020570-3b592600-6c5c-11e9-9534-9b0677042942.PNG)

However, 
## I failed to create a container and retrieve the parameter of the container parameter and make container capacity dynamic.

![vehicleParameter](https://user-images.githubusercontent.com/16321107/57020695-b28eba00-6c5c-11e9-86e8-8dd6a0f7bc95.PNG)

### my_containers.xml
```
<additional xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/additional_file.xsd">
 <containerStop id="containerStop1"
 	lane= "-537706053#2_0"
 	startPos = "5"
 	endPos = "25"/>
</additional>
```

## map.rou.xml
```
<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/routes_file.xsd">

    <vType id="truck" maxSpeed="1" color="1,1,0" vClass="delivery" containerCapacity="3"/>
    <vehicle id="flow0.0" type="truck" depart="40.0" >

        <route edges="307244665#2 307244665#3 307244665#4 -298597678#5 -298597678#4 -298597678#3 -298597678#2 -298597678#1 -298597678#0 -298597679#4 -298597679#3 -298597679#2 -298597679#1 -298597679#0 -413131917 -186109367#5 -186109367#4 -186109367#3 -186109367#2 -186109367#1 -186109367#0 -316267745#1 -316267745#0 667223560#0 667223560#1 41389171 -496261701#2 -496261701#1 -496261701#0 -41389179 413128683 413128684 41389178 160253728 107538178#0 31794904#0 31794904#1 31794904#2 31794904#3 31794904#4 31794904#5 31794904#6 496332196#0 496332196#1"/>
    </vehicle>
</routes>
```

Therefore, I checked the paper named [INTELLIGENT TRANSPORTATION SIMULATION FOR SUPPORTING SMART CITY FOOD DISTRIBUTION](http://eprints.utar.edu.my/2260/1/CN-2016-13ACB08246-1.pdf), it showed the reply from contributor to the retrieval problem in 2016.

![reply](https://user-images.githubusercontent.com/16321107/57021054-c25ace00-6c5d-11e9-8e1a-c4648e90f306.PNG)

##However,
I saw another paper named [Extensions for logistic and public transport in SUMO](QBAJ&pg=PA83&lpg=PA83&dq=Extensions+for+logistic+and+public+transport+in+SUMO&source=bl&ots=vA8UuKtJ2D&sig=ACfU3U045Iy5l6H2_rbaEHC4NeFMm8qPKA&hl=zh-TW&sa=X&ved=2ahUKEwjamb3ewfrhAhUU9LwKHQs0DKIQ6AEwCHoECAgQAQ#v=onepage&q=Extensions%20for%20logistic%20and%20public%20transport%20in%20SUMO&f=false) from **3rd SUMO Conference 2015 Berlin, Germany**.

To sum up, 

## how can I add containers to my set-up location and load/unload the containers like the figure?

![2015 sumo conference](https://user-images.githubusercontent.com/16321107/57021403-ac014200-6c5e-11e9-853e-96dfca74b860.PNG)

**Comment by namdre** [comment]
Containers are very similar to persons. Basically, you translate
'person' -> 'container'
'walk' -> tranship
'ride' -> transport.

With some limitations: 
- TraCI still doesn't support containers.
- containerFlow isn't yet implemented.
- the visualisation for containers that are waiting at a containerStop and for containers being transported are not as advanced as those for persons.

**Comment by YenKang** [comment]
Sehr geehreter Her Erdmann,
I was appreciated to your explicit reply, thus I would decide to use "person" as my container.
Futhermore, I also realize your translation from the command of container to person in paper of **2015 SUMO conference**.

![55](https://user-images.githubusercontent.com/16321107/57024588-8c224c00-6c67-11e9-9a85-ad50df47d741.PNG)

 mit freundlichen Grüßen

Kang Yen
